project/Build.scala

```diff
@@ -129,13 +129,14 @@ object DottyBuild extends Build {
     settings(publishing)
 
 
-  lazy val `dotty-interfaces` = project.in(file("interfaces")).
+  lazy val `dotty-interfaces`: Project = project.in(file("interfaces")).
     settings(sourceStructure).
+    dependsOn(`dotty-library`).
     settings(
       // Do not append Scala versions to the generated artifacts
       crossPaths := false,
       // Do not depend on the Scala library
-      autoScalaLibrary := false,
```

Comment:
We shouldn't do this either. Now you're tying `dotty-interfaces` to a particular version of `scala-library` even though `scala-library` is not necessary to use `dotty-interfaces`, only to compile it.